### PR TITLE
Update Dshot Bidir autodefaults and dynamic notch options

### PR DESCRIPTION
**Pull request by Asizon**
Removed Bandwitdh option and added again notch_Q to go with https://github.com/betaflight/betaflight/pull/10767
Also updated Dshot Bidir Autodefaults feature added for 10.8.0 in https://github.com/betaflight/betaflight-configurator/pull/1955

image

### CODE REVIEW

**Comment by Asizon** [comment]
Testers can use these artifacts:

https://dev.azure.com/Betaflight/Betaflight%20Nightlies/_build/results?buildId=2870&view=artifacts&pathAsName=false&type=publishedArtifacts

**Comment by Asizon** [comment]
I need to add some Msp workarounds to this code jet.

**Comment by Asizon** [comment]
Ready for test and review :)

**Review by haslinghuis — APPROVED** [review]

File: src/js/tabs/motors.js
Comment (McGiverGim) [inline src/js/tabs/motors.js:665]: Indent the second line.
```suggestion
                const dynFilterNeedChange = (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) ? (FC.FILTER_CONFIG.dyn_notch_count !== self.previousFilterDynCount) :
                    (FC.FILTER_CONFIG.dyn_notch_width_percent !== self.previousFilterDynWidth);

```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/js/msp/MSPHelper.js
Comment (haslinghuis) [inline src/js/msp/MSPHelper.js:2075]:
```suggestion
                    buffer
                    .push8(FC.FILTER_CONFIG.dyn_lpf_curve_expo)
                    .push8(FC.FILTER_CONFIG.dyn_notch_count)
                    .push16(FC.FILTER_CONFIG.dyn_notch_bandwidth_hz);
                    .push8(FC.FILTER_CONFIG.dyn_notch_count);

```

Comment (Asizon) [reply src/js/msp/MSPHelper.js:2075]: this looks good @haslinghuis but this does not follow the coding style we have followed so far in msp helper.

Comment (McGiverGim) [reply src/js/msp/MSPHelper.js:2075]: True, I prefer the old way...

**Review by McGiverGim — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Comment by haslinghuis** [comment]
Cannot merge or test before https://github.com/betaflight/betaflight/pull/10767 is merged

**Comment by blckmn** [comment]
AUTOMERGE: (fail)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:33:11) -> pass
 - commit count less or equal to three (1 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> fail
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> fail
 - assigned to an approver (Asizon) -> pass
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:44:58) -> pass
 - commit count less or equal to three (1 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> fail
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> fail
 - assigned to an approver (Asizon) -> pass
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:48:12) -> pass
 - commit count less or equal to three (1 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> fail
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> fail
 - assigned to an approver (Asizon) -> pass
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed (10/06/2021 00:52:04) -> PASS
 - commit count less or equal to three (1 commits) -> PASS
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> FAIL
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver (Asizon) -> PASS
 - approver count at least three (3 approvers) -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by haslinghuis** [comment]
For testing this PR we need these hexfiles:

[betaflight_4.3.0_STM32G47X_0d71c4007.zip](https://github.com/betaflight/betaflight-configurator/files/6681417/betaflight_4.3.0_STM32G47X_0d71c4007.zip)
[betaflight_4.3.0_STM32F745_0d71c4007.zip](https://github.com/betaflight/betaflight-configurator/files/6681418/betaflight_4.3.0_STM32F745_0d71c4007.zip)
[betaflight_4.3.0_STM32F7X2_0d71c4007.zip](https://github.com/betaflight/betaflight-configurator/files/6681419/betaflight_4.3.0_STM32F7X2_0d71c4007.zip)
[betaflight_4.3.0_STM32F411_0d71c4007.zip](https://github.com/betaflight/betaflight-configurator/files/6681420/betaflight_4.3.0_STM32F411_0d71c4007.zip)

**Comment by haslinghuis** [comment]
Tested on Matek F722 with above firmware. Where is the Gyro RPM Filter part?
Screenshot from 2021-06-20 03-20-30

Hmm. after pasting in my diff it shows up:
Screenshot from 2021-06-20 03-23-40

My diff
```
# diff

# version
# Betaflight / STM32F7X2 (S7X2) 4.2.9 Apr 27 2021 / 19:34:29 (e097f4ab7) MSP API: 1.43
# config: manufacturer_id: MTKS, board_name: MATEKF722MINI, version: f173f9bc, date: 2019-10-02T14:43:13Z

# start the command batch
batch start

board_name MATEKF722MINI
manufacturer_id MTKS

# name: MF722

# feature
feature -AIRMODE

# beeper
beeper -ARMING_GPS_FIX
beeper -GPS_STATUS
beeper -BLACKBOX_ERASE
beeper -CAM_CONNECTION_OPEN
beeper -CAM_CONNECTION_CLOSE

# beacon
beacon RX_SET

# map
map TAER1234

# serial
serial 20 1 230400 57600 0 115200

# aux
aux 0 0 0 1950 2050 0 0
aux 1 1 1 1950 2050 0 0
aux 2 2 1 1400 1600 0 0
aux 3 13 2 1950 2050 0 0
aux 4 35 2 1950 2050 0 0

# rxrange
rxrange 0 988 2011
rxrange 1 988 2011
rxrange 2 988 2011
rxrange 3 988 2011

# master
set acc_calibration = 455,30,189,1
set serialrx_provider = CRSF
set blackbox_device = NONE
set dshot_bidir = ON
set motor_poles = 12
set name = MF722
set display_name = Betaflight

profile 0

rateprofile 0

# end the command batch
batch end

# 
```

**Comment by Asizon** [comment]
RPM filters group on Filters tab shows when you enable Dshot Bidir function and hides if it is disabled :)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - Github identifies PR as mergeable -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
